runtime/interface.h

```diff
@@ -243,6 +243,11 @@ inline void f$kphp_turn_on_host_tag_in_inner_statshouse_metrics_toggle() {
   StatsHouseManager::get().turn_on_host_tag_toggle();
 }
 
+template <typename F>
+inline void f$kphp_extended_instance_cache_metrics_init(F &&callback) {
+  StatsHouseManager::get().set_normalization_function(normalization_function{std::forward<F>(callback)});
```

Comment:
we need to add critical section here to guard possible heap usage in `std::function` constructor